Snakefile-sorting-hat

```diff
@@ -187,7 +187,7 @@ rule produce_recalculations_call:
         pushd {input.working_directory}
 
 
-        snakemake --latency-wait 100 {config['sm_options']} --config \
+        snakemake --restart-times 4 --latency-wait 100 {config['sm_options']} --config \
```

Comment:
No obligation, but it can be convenient to handle dynamic assignment of memory based on the number of `attempts` here: 
```
def get_mem_mb(wildcards, attempt):
     """
     To adjust resources in rule baseline_coexpression.
     """
     return (2**attempt) * 4000
```